test/support/data_case.exs

```diff
@@ -2,51 +2,21 @@ Code.require_file("test_keyspace.exs", __DIR__)
 
 defmodule Cassandrax.DataCase do
   use ExUnit.CaseTemplate
-
   alias Cassandrax.TestConn
 
   using do
     quote do
       setup_all do
-        start_test_connection()
-        :ok
-      end
-
-      defp start_test_connection do
         hostname = System.get_env("CASSANDRA_HOST") || "127.0.0.1"
-        port = System.get_env("CASSANDRA_PORT") || "9043"
-
-        test_conn_attrs = [
-          nodes: ["#{hostname}:#{port}"],
-          protocol_version: :v4,
-          username: "cassandra",
-          password: "cassandra"
```

Comment:
Where did all this go? I wouldn't remove test username and password, it's useful to ensure `Cassandrax` is able to use them to authenticate.

Reply:
brought back on [76cdef1](https://github.com/loopsocial/cassandrax/pull/26/commits/76cdef15cf7b3617a9c2cd2a761e2289155e2593)